normalize_repos/log.py

```diff
@@ -59,12 +51,11 @@ def update_format(self, record):
         """
 
         prefix = "\u001b["
-        color = f"{self.color_map[record.levelno]}m"
         bold = "1m"
         reset = "0m"
         self._style._fmt = (
+            "::%(levelname)-8s file=%(filename)-20s,line=%(lineno)-4d::"
```

Comment:
Using `levelname` yields the level in capital letters. GitHub will not accept `SUCCESS`, `WARN`, `INFO` etc. Only `debug`, `warning` and `error` are handled by [GitHub Workflows](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions).

Reply:
Ya, it does yield level in capital letters but I noticed this line in the link you provided. 

> Note: Workflow command and parameter names are not case-sensitive.

Reply:
Do we need to change the format only for `debug,` `warning` and `error` levels?

Reply:
Yes please apply formatting only for those levels. I'm okay with uppercase letters if the commands are case-sensitive.